### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by nikoescobal**
## In this PR, I did the following:
- Project setup
- Configure Routing
- Create the basic structure for React and Redux files
- Enable switching between Home and Categories pages

### CODE REVIEW

File: src/pages/Categories.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Categories() {
+  return <div>Under Construction</div>;
+}
+
+export default Categories;
```
Comment (dasileker) [inline src/pages/Categories.js:7]: - [ ] Please relocate the file to components folders. no need for a page folder.

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <div>
+      Home
+    </div>
+  );
+}
```
Comment (dasileker) [inline src/pages/Home.js]: - [ ] also for this file relocate this.

File: src/redux/reducers/bookReducer/book.reducer.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      chapter: 'Chapter 17',
+    },
+    {
+      genre: 'Fiction',
+      title: 'The Alchemist',
+      author: 'Paulo Coelho',
+      completed: 88,
+      chapter: 'Chapter 7',
+    },
+  ],
```
Comment (dasileker) [inline src/redux/reducers/bookReducer/book.reducer.js]: - [ ] this is good but you need this one in your components and then call the prop in your booklist

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @nikoescobal,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- You have started a good start and you need just to relocate the files + add the form input for the book.

Capture web_13-11-2021_131631_gitlab com


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @nikoescobal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**